drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -640,7 +640,7 @@ static uint8_t nrf5_get_acc(const struct device *dev)
 {
 	ARG_UNUSED(dev);
 
-	return CONFIG_IEEE802154_NRF5_DELAY_TRX_ACC;
```

Comment:
This orphanes IEEE802154_NRF5_DELAY_TRX_ACC in Kconfig.nrf5, correct? I cannot see any other instance where it's used. Should be deprecated or removed then.